examples/slider/slider-1.html

```diff
@@ -25,81 +25,209 @@
     </ul>
   </nav>
   <main>
-    <h1>Horizontal Slider Example</h1>
+    <h1>Color Viewer Slider Example Using SVG</h1>
     <p>
-      Following is an example of a color picker that demonstrates the 
+      Following is an example of a color viewer that demonstrates the
       <a href="../../#slider">slider design pattern.</a>
-      Change the background color of the box below the picker by adjusting the sliders for red, green, and blue values.
-      The HEX and RGB values of the chosen color are displayed by the color box.
+      Change the background of the color view box by adjusting the sliders for red, green, and blue values.
+      The HEX and RGB values of the chosen color are displayed by the color view box.
     </p>
     <p>Similar examples include: </p>
     <ul>
       <li>
-        <a href="slider-2.html">Slider Examples with aria-orientation and aria-valuetext</a>: 
+        <a href="slider-2.html">Slider Examples with aria-orientation and aria-valuetext</a>:
         Three thermostat control sliders that demonstrate using aria-orientation and aria-valuetext.
       </li>
     </ul>
     <section>
        <h2 id="ex_label">Example</h2>
-          <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>        
+          <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
           <div id="ex1">
-            <div id="idSliders">
-              <div id="idRed" class="label">Red</div>
-              <div class="aria-widget-slider">
-                <div class="rail" style="width: 300px;">
-                  <div id="idRedValue"
-                       role="slider"
-                       tabindex="0"
-                       class="thumb"
-                       aria-valuemin="0"
-                       aria-valuenow="0"
-                       aria-valuemax="255"
-                       aria-labelledby="idRed"></div>
+            <div class="color-picker-sliders" role="group" aria-labelledby="id-color-picker">
+              <h2 id="id-color-picker">Color Viewer</h2>
+              <div id="id-red" class="label">Red</div>
+              <div class="color-group red">
+                <button class="change dec10" aria-label="decrement red ten" tabindex="-1">
```

Comment:
Would aria-label be better as "decrease red by 10"? (and similar for all the others?)